src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.username=rooddst
-spring.datasource.password=Passcfcsworda
+spring.datasource.username=root
+spring.datasource.password=Password
```

Comment:
It's really bad practice to use the root user; we should agree on something else, or the trainers are definitely going to point this out and tell us we should be using something else